spec.html

```diff
@@ -39257,14 +39219,13 @@ <h1>prototype</h1>
   <emu-clause id="sec-generator-objects">
     <h1>Generator Objects</h1>
     <p>A Generator object is an instance of a generator function and conforms to both the <i>Iterator</i> and <i>Iterable</i> interfaces.</p>
-    <p>Generator instances directly inherit properties from the object that is the value of the *"prototype"* property of the Generator function that created the instance. Generator instances indirectly inherit properties from the Generator Prototype intrinsic, %Generator.prototype%.</p>
+    <p>Generator instances directly inherit properties from the object that is the initial value of the *"prototype"* property of the Generator function that created the instance. Generator instances indirectly inherit properties from the Generator Prototype intrinsic, %GeneratorFunction.prototype.prototype%.</p>
 
     <emu-clause id="sec-properties-of-generator-prototype">
       <h1>Properties of the Generator Prototype Object</h1>
       <p>The Generator prototype object:</p>
       <ul>
-        <li>is <dfn>%GeneratorPrototype%</dfn>.</li>
-        <li>is <dfn>%Generator.prototype%</dfn> (<dfn>%GeneratorFunction.prototype.prototype%</dfn>).</li>
+        <li>is the initial value of the *"prototype"* property of %GeneratorFunction.prototype%.</li>
```

Comment:
```suggestion
        <li>is <dfn>%GeneratorFunction.prototype.prototype%</dfn>.</li>
```

Reply:
This isn't needed, because it's not a top-level intrinsic.

Reply:
It’s necessary for auto‑linking though: <https://github.com/tc39/ecma262/pull/2041>

Reply:
let's fix that in #2041 instead